tests/export.spec.ts

```diff
@@ -92,6 +96,104 @@ test('export html contain <blockquote>', async ({ page }) => {
 
   const htmlText = await export2Html(page);
   expect(htmlText).toContain(
-    '<header><h1 class="page-title"></h1></header><div><blockquote class="quote">page quote is here</blockquote></div>'
+    '<header><h1 class="page-title">Untitled</h1></header><div><blockquote class="quote">page quote is here</blockquote></div>'
+  );
+});
+
+test('export markdown title', async ({ page }) => {
+  await enterPlaygroundRoom(page);
+  await initEmptyEdgelessState(page);
+  await focusTitle(page);
+  const markdownText1 = await export2markdown(page);
+  expect(markdownText1).toEqual('# Untitled');
+
+  await type(page, 'this is title');
+  const markdownText2 = await export2markdown(page);
+  expect(markdownText2).toEqual('# this is title');
+});
+
+test('export markdown list', async ({ page }) => {
```

Comment:
This is very inefficient. We should migrate to calling `page` APIs for initializing doc content directly afterward.